### ISSUE TITLE
Project Carousel

### ISSUE THREAD

**Issue by j0hannr** [studio-arrenberg/quartiersplattform#103]
## Project Carousel
Projects are listed in the Project Carousel (see picture attached below). Currently Projects are sorted by date `modified`. 
Function `projekt_carousel()` in `functions.php`

## Desire
The logged in users post should all be at the top (`public` & `private`) and secondarly orrdered by the date `modified`.

**Comment by j0hannr** [comment]
I tested the code! Looks good :)

**There is one more thing:** Users Projects at the top should be ordered by `date: modified` as well, could you just update this.

Probably just adding

```
'orderby' => 'modified',
```

**Comment by healer-1205** [comment]
@j0hannr 
I made a new array to display user projects at the top.
and as u can see in my code, i add some functionalities for that.
add 'orderby' => 'modified'

**Comment by j0hannr** [comment]
Yes, but it does not order the user Projects at the top. I have commented to your PR

**Comment by proweb95** [comment]
@tomas_1205 (+856 20 59 191 840) Thomas, you lied to me, when will the payout be?
